src/transaction/flashback.h

```diff
@@ -108,6 +109,8 @@ typedef struct flashback_summary_context
 extern int flashback_make_summary_list (THREAD_ENTRY * thread_p, FLASHBACK_SUMMARY_CONTEXT * context);
 extern void flashback_cleanup (THREAD_ENTRY * thread_p, FLASHBACK_SUMMARY_CONTEXT * context);
 
+extern bool flashback_is_duplicated_request (THREAD_ENTRY * thread_p);
+extern void flashback_initialize (THREAD_ENTRY * thread_p);
```

Comment:
Is `flashback_init()` better to keep name consistency?

And is `flashback_initialize()` function which initializes the module?

If it is initializing module, I usually watched the name as `[module_name]_init()` that the function of name executes to initialize the module.

I executed '**grep -r [keyword] ***' and then rarely its name is 'initialize()'. 

Actually, it is **not** related with **functional view**. It is only about the name **consistency**.

what do you think?

Briefly,
**AS-IS**
flashback_initialize()

**TO-BE**
flashback_init()

Reply:
Thanks for feedback. 
'init' is just an abbreviation for 'initialize'. 
And there seems to be no consistency in the function names related to 'init' or 'initialize' in legacy code.
If it is required to be consistent, perfmon_initialize, locator_initialize, etc. should be modified too.

Reply:
Oh I see thank you for replying.
I knew the name consistency should be kept. 
Thank you.